presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/util/CassandraCqlUtils.java

```diff
@@ -48,12 +49,12 @@ public static String validColumnName(String identifier)
             return "\"\"";
         }
 
-        return quoteIdentifier(identifier);
+        return quote(identifier);
     }
 
-    private static String quoteIdentifier(String identifier)
+    public static String escapeSingleQuote(String identifier)
     {
-        return '"' + identifier + '"';
+        return identifier.replace("'", "''");
```

Comment:
I am concerned about escape function that doesn't quote.
Generally, escaping and quoting should be done at the same time, so they should be done
at once, in one function.